Proteomics/Modifications/ModificationGeneral.cs

```diff
@@ -26,18 +39,19 @@ public class ModificationGeneral
 
         public bool ValidModification
         {
-            get { return (this.Id != null && (this.ChemicalFormula != null || this.MonoisotopicMass != null) && this.Position != null && this.ModificationType != null); }
+            get { return (this.Id != null && (this.ChemicalFormula != null || this.MonoisotopicMass != null) && this.Position != "Unassigned." && this.ModificationType != null && this.FeatureType != "CROSSLINK"); }
             private set { ValidModification = value; }
```

Comment:
can you remove this `private set` or does visual studio complain?

Reply:
it does not complain. I don't think it should be set publicaly. maybe i'm understanding it wrong.

Reply:
I think you're not ever setting it. Every time you're Getting it, it reevaluates whether it's valid or not and returns that, without storing the value. The evaluation not very computationally expensive so I think this is fine. If you remove the Set then there is no way of setting it, which is fine, because you're never actually trying to store it

Reply:
i removed the set. i didn't know i could do that. good to knwo.